### PR TITLE
Add support for basic authentication

### PR DESCRIPTION
**Pull request by rolfvandekrol**

### CODE REVIEW

File: config/elasticsearch.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 return [
     'host' => env('ELASTICSEARCH_HOST', 'localhost:9200'),
+    'username' => env('ELASTICSEARCH_USERNAME', null),
+    'password' => env('ELASTICSEARCH_PASSWORD', null),
```
Comment (JaZo) [inline config/elasticsearch.php]: `null` is default, so you don't have to provide it as fallback.

**Review by JaZo — CHANGES_REQUESTED** [review]

File: config/elasticsearch.php (hunk 1 above)
Comment (rolfvandekrol) [reply config/elasticsearch.php]: yeah, i know, but in this case i prefer it being explicit. your call though :)

Comment (JaZo) [reply config/elasticsearch.php]: It is uncommon to explicitly provide `null` as fallback and it even gets removed in the scaffold itself, see https://github.com/laravel/laravel/pull/5791 for example.

**Review by JaZo — APPROVED** [review]

**Pull request merged**